content/administration/upgrade.rst

```diff
@@ -1,264 +1,314 @@
 
-Testing
-=======
 
-This phase allows you to review an upgraded version of your database without affecting your
-production database in any way. We suggest that you run the test upgrade process at least once, but
-you can do it as many times as you need (one at a time).
+.. _upgrade/request-test-database:
 
-Once you receive your upgraded test database, check that all data, processes, and functionality are
-still correct and working as expected.
+Obtaining an upgraded test database
+-----------------------------------
 
-If you do find discrepancies, :ref:`report your issues <upgrade/test-assistance>` and :ref:`request
-a new test database <upgrade/test-db-request>` when the reported issues are fixed in the upgrade
-script.
+The `Upgrade website <https://upgrade.odoo.com/>`_ is the main platform for requesting an upgraded database but depending on your hosting, you are be able to upgrade your database from the command line, from `<https://odoo.com/my/databases>`_, or from your `Odoo SH project <https//odoo.sh/project>`_.
 
-If you do not find any discrepancies, you can move on to the upgrade of your production database.
+In all cases, your data follows the same process as described in the following diagram:
 
-.. important::
-   A test database is only intended for testing and remains completely unrelated to your present or
-   future production database. Any data you add, or changes you make, will not be reflected in your
-   upgraded production database.
+.. image:: /administration/upgrade/schema_upgrade_process.png
+    :width: 50%
+    :alt: Schema of the upgrade process : Export the database to a file, upload to the upgrade server, standard upgrade scripts, download the upgraded database, import the file into a database, (Optional : Custom upgrade scripts)
+    :align: center
+
+Where the standard upgrade scripts consists of a sequence of :ref:`reference/upgrade/migration-scripts`, each responsible for upgrading a specific part of a module's data.
 
 .. note::
-   Test databases are neutered and features are disabled to prevent them from having an impact on
-   the production database:
+    You can always request a new upgraded test database in case you want to test the upgrade again or if you want to test the upgrade to a different version of Odoo.
+
+.. tabs::
+
+    .. group-tab:: Odoo Online
+
+        .. _upgrade/request-test-database/odoo-online:
+
+        Odoo databases can be manually upgraded directly from the main Odoo website. To upgrade an Odoo database, navigate to the `database manager page <https://www.odoo.com/my/databases>`_ page and sign in.
+
+        The database manager page displays all of the Odoo databases associated with the user's account. Any databases that are not already on the most recent version of Odoo display an **arrow in a circle** icon next to the database name, indicating that the database can be upgraded.
+
+        .. image:: /administration/upgrade/odoo_online/databases-page.png
```

Comment:
I included your screenshot but I am scared that it does not display enough information, like where exactly is that screenshot supposed to show on the page, since there is no context surrounding it. It is still better than the previous one though so i'll include it